Rajitha/recursion-practice.js

```diff
@@ -0,0 +1,53 @@
+/****************************************************
+* Create a function to get the factoral of a number.
+* Input: factoral(5)  Output: 120
+****************************************************/
+var factorial = function(number) {
+  if (number <= 0) {
```

Comment:
This still works because anything multipled by 1 is itself but ideally the end case should be `<=1`